x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/IndexLifecycleExplainResponse.java

```diff
@@ -35,6 +35,8 @@
     private static final ParseField ACTION_FIELD = new ParseField("action");
     private static final ParseField STEP_FIELD = new ParseField("step");
     private static final ParseField FAILED_STEP_FIELD = new ParseField("failed_step");
+    private static final ParseField IS_TRANSITIVE_ERROR_FIELD = new ParseField("is_transitive_error");
```

Comment:
I'm not sure about this name, I think a better name might be `error_auto_retryable` or something similar about how the error *will* be retried.

Reply:
Transient?